### ISSUE TITLE
Inference code

### ISSUE THREAD

**Issue by mf0212** [nahyeonkaty/textboost#1]
do you have already inferenc code?

**Comment by nahyeonkaty** [comment]
Yes, you can find the inference code in evaluate_dreambooth.py. Please refer to the README for specific usage.

**Comment by nahyeonkaty** [comment]
Yes, we have added inference.py, you can also check its usage through README.

**Comment by nahyeonkaty** [comment]
Yes, we have added inference.py. You can also check its usage in the README.

**Comment by mf0212** [comment]
I'm very appropriate to your work, I have run successfully with inference code, but I tried to fine-tune with my own image(example: Ronaldo image), with init_word=<ronaldo>, but I check that the result is not in high fidelity. I have checked that your model is trained on 30 instances (which don't include my instance), so if I want to train on a new instance, it will be in high performance. If not can you have any recommendations for me to apply my own images?

**Comment by nahyeonkaty** [comment]
Setting the initializer_token properly should work. You can also try setting the class_token and instance_token appropriately.

**Comment by nahyeonkaty** [comment]
Setting the initializer_token properly should work. You can also try setting the class_token and instance_token appropriately. If these steps don't resolve your case, please email me, and I'll try to assist as time allows.
